### PR TITLE
Use latest worker

### PR DESCRIPTION
**Pull request by karaxuna**
Use worker instead of sync package. Use latest core.

---

Change-type: major

### CODE REVIEW

**Comment by karaxuna** [comment]
@balena-ci retest

**Comment by joshbwlng** [comment]
@balena-ci rebase

File: test/integration/balena-api-translate.spec.ts [hunk 1]
```diff
@@ -49,32 +47,75 @@ async function prepareEvent(event: any): Promise<any> {
...
+						offset:
+							_.findIndex(testCase.steps, _.first(variation.combination)) + 1,
+						headIndex: testCase.headIndex || 0,
+						original: testCase.steps,
+
+						// If we miss events such as when a head card was archived,
+						// we usually can't know the date this happened, but we can
+						// still apply it with a date approximation. In those cases,
+						// its helpful to omit the update events from the tail checks.
+						ignoreUpdateEvents: true /*!_.isEqual(
```
Comment (karaxuna) [inline test/integration/balena-api-translate.spec.ts:95]: Was getting unnecessary update event on `data.avatar` field. @joshbwlng any idea why?

**Comment by karaxuna** [comment]
@balena-ci I self certify!

**Pull request merged**